sources/packages/backend/apps/workers/src/controllers/assessment/_tests_/e2e/assessment.controller-verifyAssessmentCalculationOrder.e2e-spec.ts

```diff
@@ -969,9 +970,133 @@ describe("AssessmentController(e2e)-verifyAssessmentCalculationOrder", () => {
     expect(FakeWorkerJobResult.getResultType(result)).toBe(
       MockedZeebeJobResult.Complete,
     );
-    // The result will not have data from SFAS or SFAS part time application data.
+    // The result will not have data from SFAS or SFAS part-time application data.
     expect(FakeWorkerJobResult.getOutputVariables(result)).toStrictEqual({
       isReadyForCalculation: true,
+      // Student contribution total.
+      programYearTotalFederalFSC: 0,
+      programYearTotalProvincialFSC: 0,
+      programYearTotalScholarshipsBursaries: 0,
+      programYearTotalSpouseContributionWeeks: 0,
```

Comment:
Same question for a later comment, should we return 0 when values are not present?

Reply:
we are not doing any calculation still in the workflow, i thought it would be better to return 0 than null, as the variable name states "programYearTotal" either it can be 0 or some numeric value.

Reply:
My point is the variable `programYearTotal*` will not be even there if not present.